tests/test_document.py

```diff
@@ -102,7 +103,7 @@ def test_document_validate_failures_returns_informative_messages(self):
             'Package declared license must be instance of spdx.utils.SPDXNone '
             'or spdx.utils.NoAssert or spdx.document.License'
         ]
-        assert expected == messages
+        assert sorted(expected) == sorted(messages)
```

Comment:
I wonder why the results order would not be stable?

Reply:
In all my tests the result order was always stable, but I guess it does not hurt to add `sorted` here...